investment-time/README.md

```diff
@@ -0,0 +1,81 @@
+# Investment Time
+
+This directory contains documentation of DataMade's implementation of investment
+time, as well as pointers to documents related to the proposal, pilot, and
+adoption of investment time.
+
+## Contents
+
+- README – You are here! 🎉
+- [Research](./research.md)
+
+## Investment Time at DataMade
+
+### How much investment time do I have?
+
+DataMade employees may use 10% of hours worked in a given two-week cycle, or
+one of every 10 days worked, for investment time.
+
+Employees will be assigned to one of two cohorts with alternating two-week
+cycles, so that investment time is staggered across the team. Assignments will
+appear on the DataMade calendar as `${STAFF} investment time`, every other
+Friday.
+
+Staff may use investment time as their schedule allows during the week of their
+designated investment day, however it may not be used during their off week.
+Investment time should be added to DataMade's weekly planning document at the
+start of each week.
+
+### How can investment time be used?
+
+Developers may use investment time for self-directed work to improve their
+skills within the existing DataMade stack. Some example uses include:
+
+- Completing and online course or tutorial
+- Retrofitting an existing application to use a different stack, e.g.,
+implementing a Django site in Gatsby
+- Working on a new, non-client project that uses some part/s of the DataMade
+stack, in consultation with their primary manager and potentially a partner
+- Working on an R&D issue delegated by a lead developer
+
+For lead developers, investment time encompasses all of the above, plus R&D, as
+documented in [CONTRIBUTING.md](../CONTRIBUTING.md).
+
+Developers should discuss their plans for investment time with their primary
+managers during their regular 1:1. Managers can help to shape a proposal or come
+up with an idea.
+
+Investment time should produce some artifact, be it code, a document, or notes
+from a meeting. These should be published in a place accessible to the entire
+team and remain accessible in perpetuity, e.g., in a Google Document or GitHub
+repository.
+
+#### Investment time and client work
```

Comment:
@derekeder @fgregg I adapted my recommendation based on the conversation we had during HR. Your feedback on this section is much appreciated!

Reply:
@hancush this looks good to me and covers what we discussed at HR. I left one comment about improvements that reduce maintenance overhead as something that is a strong justification for getting a client to pay for it